solutions/observability/logs/streams/management/advanced.md

```diff
@@ -0,0 +1,10 @@
+---
+applies_to:
+    serverless: preview
+---
+# Configure advanced settings [streams-advanced-settings]
+
+The **Advanced** tab on the **Management** page shows the lower-level details of your stream. While Streams attempts to make things as easy as possible, it doesn't currently support every change. From the **Advanced** tab, you can manually interact with the index or component templates, or modify any of the other ingest pipelines that are being used.
```

Comment:
> While Streams attempts to make things as easy as possible, it doesn't currently support every change.

I'm not exactly sure what this means. Can we be more specific? "While Streams simplifies many configurations, it doesn't currently support all pipeline or template modifications." or something like that?